nimare/meta/utils.py

```diff
@@ -397,7 +397,7 @@ def _convolve_sphere(kernel, peaks):
     return kernel_data
 
 
-def compute_ale_ma(shape, ijk, kernel):
+def compute_ale_ma(mask, ijks, exp_idx, sample_sizes, kernels, use_dict):
```

Comment:
I'd like to compare this to https://github.com/tsalo/NiMARE/blob/49191a0a4b10a2ac522828c9e558f279d5c6d6ea/nimare/meta/utils.py#L400-L438 before we merge.